poetry.lock

```diff
@@ -1,5 +1,142 @@
 # This file is automatically @generated by Poetry 1.7.0 and should not be changed by hand.
 
+[[package]]
+name = "aiohappyeyeballs"
```

Comment:
Did not check this file, but where are the new dependencies coming from?

Reply:
Obvious upgrades are `torch@2.4.1` + `torch-geometric@2.6.1`, those should imply the new dependencies.

`aiohappyeyeballs` for instance comes as a dependency of `aiohttp` (see), which is a dependency of `torch-geometric` ([see](https://github.com/promaton/torch-points3d/blob/bugra/torch24-cuda121-upgrade/poetry.lock#L1325))